### ISSUE TITLE
cURL error 7: Failed to connect to namesite.com port 9980: Connection refused

### ISSUE THREAD

**Issue by caygri** [owncloud/richdocuments#220]
Hello,

i would like to use your app on my owncloud but i have this error:

`cURL error 7: Failed to connect to namesite.com port 9980: Connection refused`

I also open the port 9980.

`To                         Action      From
--                         ------      ----
9980/tcp                   ALLOW IN    Anywhere
80                         ALLOW IN    Anywhere
9980                       ALLOW IN    Anywhere
9980/tcp (v6)              ALLOW IN    Anywhere (v6)
80 (v6)                    ALLOW IN    Anywhere (v6)
9980 (v6)                  ALLOW IN    Anywhere (v6)`

Maybe the problem is i havent a subcription on collabora?

**Comment by caygri** [comment]
no, i try to install and i have this error

`W: Target Packages (Packages) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:6
W: Target Translations (en) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:6
W: Target Packages (Packages) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:7
W: Target Translations (en) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:7
W: Target Packages (Packages) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:8
W: Target Translations (en) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:8
E: The method driver /usr/lib/apt/methods/https could not be found.
N: Is the package apt-transport-https installed?
E: Failed to fetch https://www.collaboraoffice.com/repos/CollaboraOnline/CODE-ubuntu1804/./InRelease
E: Some index files failed to download. They have been ignored, or old ones used instead.
W: Target Packages (Packages) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:6
W: Target Translations (en) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:6
W: Target Packages (Packages) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:7
W: Target Translations (en) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:7
W: Target Packages (Packages) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:8
W: Target Translations (en) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:8`

**Comment by cr4shydlo** [comment]
How you try to install it ?

sudo docker pull collabora/code
sudo docker run -t -d -p 127.0.0.1:9980:9980 -e "domain=<your-dot-escaped-domain>" \
        -e "username=admin" -e "password=S3cRet" --restart always --cap-add MKNOD collabora/code

**Comment by caygri** [comment]
i haven't docker i follow this istruction  https://www.collaboraoffice.com/code/

but when i launch the last line i have this error

`W: Target Packages (Packages) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:6
W: Target Translations (en) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:6
W: Target Packages (Packages) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:7
W: Target Translations (en) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:7
W: Target Packages (Packages) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:8
W: Target Translations (en) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:8
E: The method driver /usr/lib/apt/methods/https could not be found.
N: Is the package apt-transport-https installed?
E: The method driver /usr/lib/apt/methods/https could not be found.
N: Is the package apt-transport-https installed?
E: Failed to fetch https://www.collaboraoffice.com/repos/CollaboraOnline/CODE-ubuntu1804/./InRelease
E: Failed to fetch https://www.collaboraoffice.com/repos/CollaboraOnline/CODE/./InRelease
E: Some index files failed to download. They have been ignored, or old ones used instead.
W: Target Packages (Packages) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:6
W: Target Translations (en) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:6
W: Target Packages (Packages) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:7
W: Target Translations (en) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:7
W: Target Packages (Packages) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:8
W: Target Translations (en) is configured multiple times in /etc/apt/sources.list:5 and /etc/apt/sources.list:8`

**Comment by caygri** [comment]
mmmm...how?

**Comment by thebearon** [comment]
Actually, those are just warnings, this is the error:
E: The method driver /usr/lib/apt/methods/https could not be found. N: Is the package apt-transport-https installed?

**Comment by caygri** [comment]
No, it wasn't i install. I repeat again, and now i think missing all this dependencies

`Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 loolwsd : Depends: libpococrypto50 but it is not installable
           Depends: libpocofoundation50 but it is not installable
           Depends: libpocojson50 but it is not installable
           Depends: libpoconet50 but it is not installable
           Depends: libpoconetssl50 but it is not installable
           Depends: libpocoutil50 but it is not installable
           Depends: libpocoxml50 but it is not installable
           Depends: libssl1.1 (>= 1.1.0) but it is not installable
           Depends: libstdc++6 (>= 6) but 5.4.0-6ubuntu1~16.04.10 is to be installed`

**Comment by thebearon** [comment]
It seems you used the Ubuntu 18.04 repository link, but you have Ubuntu 16.04 installed, is that that case? If so, remove the '-ubuntu1804' from the repository link in the respective line in /etc/apt/sources.list

**Comment by caygri** [comment]
sorry for dummie question how can i remove?

**Comment by cr4shydlo** [comment]
nano /etc/apt/sources.list (or use any other editor)
search for -ubuntu1804 and delete it.

Regards.

**Comment by caygri** [comment]
Thank you so much! Now install it, but i see the same error..

**Comment by cr4shydlo** [comment]
You install it. But did you run it ?
Check loolwsd service is running.

**Comment by caygri** [comment]
`Sep 15 09:34:28 1lcu.f.time4vps.cloud systemd[1]: loolwsd.service: Unit entered failed state.
Sep 15 09:34:28 1lcu.f.time4vps.cloud systemd[1]: loolwsd.service: Failed with result 'exit-code'.
Sep 15 09:34:28 1lcu.f.time4vps.cloud systemd[1]: loolwsd.service: Service hold-off time over, scheduling restart.
Sep 15 09:34:28 1lcu.f.time4vps.cloud systemd[1]: Stopped LibreOffice Online WebSocket Daemon.
Sep 15 09:34:28 1lcu.f.time4vps.cloud systemd[1]: loolwsd.service: Start request repeated too quickly.
Sep 15 09:34:28 1lcu.f.time4vps.cloud systemd[1]: Failed to start LibreOffice Online WebSocket Daemon.`

here the problem..
